x-team/xp-ui
# Implement responsiveness on VideoPlayer

**Release Type:** *Non-breaking feature*

Fixes https://x-team-internal.atlassian.net/browse/XP-3496

## Description

Implement responsiveness on VideoPlayer

## Checklist


## Steps to Test or Reproduce

1. Go to VideoPlayer stories
    - http://localhost:9001/?path=/story/core-components-videoplayer--basic-usage
    - http://localhost:9001/?path=/story/core-components-videoplayer-use-cases--embedded-video
    - http://localhost:9001/?path=/story/core-components-videoplayer-use-cases--embedded-with-overlay-video
1. On each of these cases, resize the viewport and observe that the video player never gets bigger than 100%, always resizes respecting the aspect ratio

## Impacted Areas in Application

- `VideoPlayer`

## Screenshots

![2020-01-15 09 44 06](https://user-images.githubusercontent.com/131859/72435309-8da38e00-377c-11ea-8632-f65af293e73e.gif)

![2020-01-15 09 44 33](https://user-images.githubusercontent.com/131859/72435310-8da38e00-377c-11ea-9a8d-8b65fde6dd15.gif)
